src/views/v2/list/event/description.php

```diff
@@ -9,13 +9,17 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.9
+ * @version TBD
  *
  * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
  *
  * @see tribe_get_event() For the format of the event object.
  */
+
+if ( empty( $event->excerpt ) ) {
+	return;
+}
 ?>
 <div class="tribe-events-calendar-list__event-description tribe-common-b2">
-	<?php echo tribe_events_get_the_excerpt( $event, wp_kses_allowed_html( 'post' ) ); ?>
+	<?php echo $event->excerpt; ?>
```

Comment:
should this have `esc_html()`?

Reply:
No, the `excerpt` is generated with a template tag that takes care of escaping, and we're actually allowing a bit of HTML. 

- https://github.com/moderntribe/the-events-calendar/pull/2727/files#diff-099cdbe9ae7ffadddaa170ce58e565fbR153
- https://github.com/moderntribe/the-events-calendar/blob/release/B19.13/src/functions/template-tags/general.php#L1450